src/sentry/api/serializers/models/organization_member.py

```diff
@@ -192,3 +193,25 @@ def serialize(
         d = super().serialize(obj, attrs, user)
         d["projects"] = attrs.get("projects", [])
         return d
+
+
+class OrganizationMemberSCIMSerializer(Serializer):  # type: ignore
+    def __init__(self, expand: Optional[Sequence[str]] = None) -> None:
+        self.expand = expand or []
```

Comment:
For `__init__`, the return type annotation should be
```python
 -> "OrganizationMemberSCIMSerializer"
```
That is, putting the class name in a string prevents the error where it's referred to before we've finished defining the class. They close that gap in Python 3.9 or 3.10, I believe.

Also, a nit: Is this `Optional` because we're expecting to be passed an explicit `None` value (perhaps generated from a JSON `null`), or only for the default value? If the latter, I'd change this to:
```python
    def __init__(self, expand: Sequence[str] = ()):
        self.expand = expand
```
(A tuple, being stateless, is suitable as a default arg, unlike `[]`.)

Reply:
i copied all these typings wholesale from a class above in the file.. not sure if this was thought about at all, it seems the changes were fairly recent https://github.com/getsentry/sentry/issues/25433

Reply:
I'd be 👍 on changing `-> None` to a string type hint in all such cases, but I'm fine on leaving `Optional[Sequence[str]] = None` alone.

Reply:
will leave this alone for now -- would love to come back to this when i get a chance though!

Reply:
Belated correction... ` -> None` is indeed correct because `__init__` is not the ctor from which the object is returned, but is called *after* construction. I think what I said might have been correct for `__new__`, but don't quote me on that either.

You *would* definitely want to put the return type hint as a string (in Python 3.6) for, e.g., a static method that is explicitly `return`ing an instance of the class.